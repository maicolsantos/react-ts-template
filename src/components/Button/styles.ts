import styled from 'styled-components'

export const ButtonContainer = styled.button`
  background: ${({ theme }) => theme.color?.primary};
`
