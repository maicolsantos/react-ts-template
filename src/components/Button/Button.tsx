import React from 'react'

import { ButtonContainer } from './styles'

export const Button = () => (
  <ButtonContainer>
    <h1>Button</h1>
  </ButtonContainer>
)
