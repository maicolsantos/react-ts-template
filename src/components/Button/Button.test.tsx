import React from 'react'
import { render } from '@testing-library/react'

import { Button } from './Button'

test('renders component Button', () => {
  const { container } = render(<Button />)
  expect(container.firstChild).toMatchSnapshot(`
    <h1>Button</h1>
  `)
})
