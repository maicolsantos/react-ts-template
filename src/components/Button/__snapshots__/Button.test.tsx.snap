// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component Button: 
    <h1>Button</h1>
   1`] = `
<h1>
  Button
</h1>
`;
